hack/e2e-internal/e2e-status.sh

```diff
@@ -32,3 +34,10 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 prepare-e2e
 
 ${KUBECTL} version
+
+# Before running tests, ensure that all pods are 'Running'. Tests can timeout
+# and fail because the test pods don't run in time. The problem is that the pods
+# that a cluster runs on startup take too long to start running, with sequential
+# Docker pulls of large images being the culprit. These startup pods block the
+# test pods from running.
+source "${KUBE_ROOT}/hack/e2e-internal/wait-pods-running.sh"
```

Comment:
Actual reason is that syntax highlighting for e2e-status.sh is broken because of the complexity of quoting/variable expansion, so I wanted to develop in a separate script so I could spot blatant syntax bugs more easily. 

Then, it seemed to be a good separation of functionality, but really it isn't necessary. Want me to pull in?